test/vstest.console.UnitTests/Internal/ConsoleLoggerTests.cs

```diff
@@ -63,10 +63,14 @@ public void InitializeShouldNotThrowExceptionIfEventsIsNotNull()
         [TestMethod]
         public void InitializeWithParametersShouldThrowExceptionIfEventsIsNull()
         {
+            var parameters = new Dictionary<string, string>
+            {
+                { "param1", "value" },
+                { DefaultLoggerParameterNames.TargetFramework , "net451"}
```

Comment:
DRY!
We should have just created a test initialize for default dictionary and then add the required parameter to that.